Source/init.cpp

```diff
@@ -229,7 +229,7 @@ void init_create_window()
 #endif
 }
 
-void MainWndProc(UINT Msg, WPARAM wParam, LPARAM lParam)
+void MainWndProc(UINT Msg)

```

Comment:
Can you just change the signature of the window procedure? *headscratch*

Reply:
Why not :shrug: 

It's derived and called as a normal function since we aren't Windows-only anymore :)